tests/Unit/Evolution/Systems/RadiationTransport/M1Grey/Test_Actions.cpp

```diff
@@ -0,0 +1,136 @@
+  const DataVector J{0.5, 0.5, 0.5, 0.5, 0.5};
+  const DataVector Hn{0.5, 0.5, 0.5, 0.5, 0.5};
+  const DataVector Hx{0.5, 0.5, 0.5, 0.5, 0.5};
+  const DataVector Hy{0.5, 0.5, 0.5, 0.5, 0.5};
+  const DataVector Hz{0.5, 0.5, 0.5, 0.5, 0.5};
+  // Closure input variables
+  // First neutrino species (set to optically thick)
+  const DataVector E0{1.0, 1.0, 1.0, 1.0, 1.0};
+  const DataVector Sx0{0.0, 0.0, 0.0, 0.0, 0.0};
+  const DataVector Sy0{0.0, 0.0, 0.0, 0.0, 0.0};
+  const DataVector Sz0{0.0, 0.0, 0.0, 0.0, 0.0};
+  // Second neutrino species (set to optically thin)
+  const DataVector E1{1.0, 1.0, 1.0, 1.0, 1.0};
+  const DataVector Sx1{0.0, 0.0, 0.0, 1.0, 1.0};
+  const DataVector Sy1{0.0, 0.0, 1.0, 0.0, 0.0};
+  const DataVector Sz1{1.0, 1.0, 0.0, 0.0, 0.0};
+
+  // Fluid and metric variables
+  const DataVector vx{0.0, 0.0, 0.0, 0.0, 0.0};
+  const DataVector vy{0.0, 0.0, 0.0, 0.0, 0.0};
+  const DataVector vz{0.0, 0.0, 0.0, 0.0, 0.0};
+  const DataVector W{1.0, 1.0, 1.0, 1.0, 1.0};
+  auto metric = make_with_value<tnsr::ii<DataVector, 3>>(x, 0.0);
+  auto inverse_metric = make_with_value<tnsr::II<DataVector, 3>>(x, 0.0);
+  get<0, 0>(metric) = 1.;
+  get<1, 1>(metric) = 1.;
+  get<2, 2>(metric) = 1.;
+  get<0, 0>(inverse_metric) = 1.;
+  get<1, 1>(inverse_metric) = 1.;
+  get<2, 2>(inverse_metric) = 1.;
+
+  tuples::get<MockDistributedObjectsTag>(dist_objects)
+      .emplace(
+          0, ActionTesting::MockDistributedObject<component>{
+                 db::create<db::AddSimpleTags<tmpl::flatten<
+                     tmpl::list<component::Closure::return_tags,
+                                component::Closure::argument_tags,
+                                ::Tags::Coordinates<3, Frame::Inertial>>>>>(
+                     Scalar<DataVector>{xi}, Scalar<DataVector>{xi}, tildeP,
+                     tildeP, Scalar<DataVector>{J}, Scalar<DataVector>{J},
+                     Scalar<DataVector>{Hn}, Scalar<DataVector>{Hn},
+                     tnsr::i<DataVector, 3, Frame::Inertial>{{{Hx, Hy, Hz}}},
+                     tnsr::i<DataVector, 3, Frame::Inertial>{{{Hx, Hy, Hz}}},
+                     Scalar<DataVector>{E0}, Scalar<DataVector>{E1},
+                     tnsr::i<DataVector, 3, Frame::Inertial>{{{Sx0, Sy0, Sz0}}},
+                     tnsr::i<DataVector, 3, Frame::Inertial>{{{Sx1, Sy1, Sz1}}},
+                     tnsr::I<DataVector, 3, Frame::Inertial>{{{vx, vy, vz}}},
+                     Scalar<DataVector>{W}, metric, inverse_metric,
+                     tnsr::I<DataVector, 3, Frame::Inertial>{{{x, y, z}}})});
+  MockRuntimeSystem runner{{}, std::move(dist_objects)};
+  auto& box = runner.template algorithms<component>()
+                  .at(0)
+                  .template get_databox<typename component::initial_databox>();
+  runner.next_action<component>(0);
+
+  // Check that first species return xi=0 (optically thick)
+  const DataVector true_xi0{0.0, 0.0, 0.0, 0.0, 0.0};
```

Comment:
We've been consistently using `expected_` instead of `true_` throughout the code.